src/data/favourites.js

```diff
@@ -0,0 +1,36 @@
+const favArray = [
+  {
+    id: "0",
+    name: "Smith & Sons",
+    description: "Mom and Pop chain serving sandwhiches",
+    weekdayOpening: "MON-FRI : 9:00 - 19:00",
+    weekendOpening: "Sat : 10:00 - 15:00",
+    address: "32 Camden Hill, Eeling, London",
+    price: "£££££",
+    image: " INSERT IMAGE HERE ",
+    category: "Food & Drink",
+  },
+  {
+    id: "1",
+    name: "Dirty Onion",
+    description: "Irish Bar, Great guines on tap. Live music on saturdays.",
+    weekdayOpening: "MON-FRI : 9:00 - 19:00",
+    weekendOpening: "Sat : 10:00 - 15:00",
+    address: "36 Lamden Hill, Eeling, London",
+    price: "£",
+    image: " INSERT IMAGE HERE ",
+    category: "Other",
+  },
+  {
+    id: "2",
+    name: "Gunners art gallery",
+    description: "I SELL ART & STUFF",
+    weekdayOpening: "MON-FRI : 9:00 - 19:00",
+    weekendOpening: "Sat : 10:00 - 15:00",
+    address: "112 Backden Hill, Eeling, London",
+    price: "£££",
+    image: " INSERT IMAGE HERE ",
+    category: "Culture",
+  },
+];
+export default favArray;
```

Comment:
I don't think this array looks quite right, could you use the array within mockUser -> favourites instead? rather than duplicating

Reply:
not in use anymore, so will delete